src/coreclr/jit/clrjit.natvis

```diff
@@ -0,0 +1,105 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!--
+Licensed to the .NET Foundation under one or more agreements.
+The .NET Foundation licenses this file to you under the MIT license.
+-->
+
+
+<AutoVisualizer xmlns="http://schemas.microsoft.com/vstudio/debugger/natvis/2010">
+
+  <Type Name="Compiler">
+    <DisplayString>{{ {info.compFullName,sb} ({info.compMethodHashPrivate,x})}}</DisplayString>
+    <StringView>info.compFullName,sb</StringView>
+  </Type>
+
+  <Type Name="BasicBlock">
+    <DisplayString Condition="bbJumpKind==BBJ_COND || bbJumpKind==BBJ_ALWAYS || bbJumpKind==BBJ_LEAVE || bbJumpKind==BBJ_EHCATCHRET || bbJumpKind==BBJ_CALLFINALLY">BB{bbNum,d}->BB{bbJumpDest->bbNum,d}; {bbJumpKind,en}</DisplayString>
+    <DisplayString>BB{bbNum,d};--- {bbJumpKind}</DisplayString>
+  </Type>
+
+  <Type Name="EHblkDsc">
+    <DisplayString>type={ebdHandlerType}</DisplayString>
+  </Type>
+
+  <!-- GenTree -->
+  <Type Name="GenTree">
+    <DisplayString>{{{gtOper,en}, {gtType,en}}}}</DisplayString>
+  </Type>
+  <Type Name="GenTreeIntCon">
+    <DisplayString>{{IntCon= {((GenTreeIntCon*)this)-&gt;gtIconVal, d}}}</DisplayString>
```

Comment:
```suggestion
    <DisplayString>{{IntCon={((GenTreeIntCon*)this)-&gt;gtIconVal, d}}}</DisplayString>
```

IMO, the spaces after the equals sign are not needed

Reply:
sure